diff --git a/node_modules/inquirer/lib/ui/baseUI.js b/node_modules/inquirer/lib/ui/baseUI.js
index 037d6dd..5eb9d62 100644
--- a/node_modules/inquirer/lib/ui/baseUI.js
+++ b/node_modules/inquirer/lib/ui/baseUI.js
@@ -53,8 +53,8 @@ class UI {
 
     // Close the readline
     this.rl.output.end();
-    this.rl.pause();
-    this.rl.close();
+    // this.rl.pause();
+    // this.rl.close();
   }
 }
 
